### DISCUSSION TITLE
Github设置启用双重验证国家/地区没有中国

### DISCUSSION THREAD

**Discussion by gallop1** [community/community#33002] [Other Feature Feedback, Questions, & Ideas]
Github设置启用双重验证国家/地区没有中国,请尽快重新添加进去。The Github settings enable dual authentication in a country/region that does not have China. Please add it again as soon as possible.

**Comment by ghostinhershell** [comment]
_Translated message:_

_Github settings enable two-factor authentication country/region without China, please re-add it as soon as possible. The Github settings enable dual authentication in a country/region that does not have China. Please add it again as soon as possible._

**Comment by xin2017338** [comment]
The Github settings enable dual authentication in a country/region that does not have China. Please add it again as soon as possible.
